python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -531,6 +533,66 @@ async def main():
     compiled_dag.teardown()
 
 
+class TestMultiChannel:
+    def test_multi_channel_one_actor(self, ray_start_regular_shared):
+        a = Actor.remote(0)
+        with InputNode() as inp:
+            dag = a.inc.bind(inp)
+            dag = a.inc.bind(dag)
+            dag = a.inc.bind(dag)
+
+        compiled_dag = dag.experimental_compile()
+        assert len(compiled_dag.actor_to_tasks) == 1
+
+        num_local_channels = 0
+        num_remote_channels = 0
+        for tasks in compiled_dag.actor_to_tasks.values():
+            assert len(tasks) == 3
+            for task in tasks:
+                assert isinstance(task.output_channel, MultiChannel)
+                if task.output_channel._local_channel:
+                    num_local_channels += 1
+                if task.output_channel._remote_channel:
+                    num_remote_channels += 1
+        assert num_local_channels == 2
```

Comment:
nit: Add a comment explaining that the two local channels are within the actor and the one remote channel is between the actor and the driver

Reply:
Added https://github.com/ray-project/ray/pull/45591/commits/3170419df1d2ca736f040c1313381fe7791f9051